### DISCUSSION TITLE
Module '"bootstrap"' has no exported member 'bootstrap'.ts(2305)

### DISCUSSION THREAD

**Discussion by reekoong** [twbs/bootstrap#34857] [Q&A]
Hello

I try to use Toasts with Angular 12 but it's can't import bootstrap

![image](https://user-images.githubusercontent.com/12757006/131535107-033f3a6e-95c8-4de0-9a8b-e792bde3fdd6.png)

**Comment by seyoon20087** [comment]
Try using the following code instead of the code noted above:
```js
import Tooltip as Tooltip from '../node_modules/bootstrap/js/dist/tooltip';
```

If you are trying to import it all, you can:
```js
import * as bootstrap from 'bootstrap';
```

See this [documentation](https://getbootstrap.com/docs/5.1/getting-started/parcel/#importing-javascript).
